BigriverBookstore-api/BigriverBookstore-api.Server.WebService/Configurations/BookConfiguration.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using BigriverBookstore_api.ServiceModel;
+using JsonApiFramework.ServiceModel.Configuration;
+
+namespace BigriverBookstore_api.WebService.Configurations
+{
+    public class BookConfiguration : ResourceTypeBuilder<Book>
+    {
+        public BookConfiguration()
+        {
+            this.Hypermedia()
+                .SetApiCollectionPathSegment("book");
+            
+            this.ResourceIdentity(x => x.BookId)
+                .SetApiType("books");
+
+            this.Attribute(x => x.Title)
+                .SetApiPropertyName("book-title");
+
+            this.Attribute(x => x.DatePublished)
+                .SetApiPropertyName("date-published");
```

Comment:
Why did you choose this naming convention (with dashes) for the exported attribute names?

Reply:
I based this naming convention on the documentation found from JsonApiFramework, but I could also remove the dashes and use uppercase for the first letter of the second word or use underline between the words.

Reply:
For me it was important to have a clear reading of the property

Reply:
Updated properties names to Camel Case.